src/jsMain/kotlin/ui/components/tabs/uploadtab/FileUploadTab.kt

```diff
@@ -65,6 +68,8 @@ class FileUploadTab : RComponent<FileUploadTabProps, FileUploadTabState>() {
                 }
             }
             fileUploadButtonBar {
+                uploadText = props.polyglot.t("upload_button")
```

Comment:
Include polyglot in fileUploadButtonBar as well and move uploadText and validateText there.